tests/bootstrap.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+use Symfony\Component\Process\Process;
+
+$minkTestServer = new Process([
+    PHP_BINARY,
+    '-S',
+    '0.0.0.0:8002',
+    '-t',
+    __DIR__ . '/../vendor/mink/driver-testsuite/web-fixtures'
+]);
+$minkTestServer->start();
```

Comment:
I find this more convenient than adding commands to the workflows and more steps in the documentation.

Any thoughts of possible downsides? Should it be documented somewhere?